testsuite/docker/kafka/config/log4j.properties

```diff
@@ -91,5 +91,5 @@ log4j.logger.state.change.logger=TRACE, stateChangeAppender
 log4j.additivity.state.change.logger=false
 
 # Access denials are logged at INFO level, change to DEBUG to also log allowed accesses
-log4j.logger.kafka.authorizer.logger=INFO, authorizerAppender
+log4j.logger.kafka.authorizer.logger=DEBUG, authorizerAppender
```

Comment:
Intentional?

Reply:
I guess we can wait and see if CI fails, and if it does revert this change.